src/js/app.jsx

```diff
@@ -53,7 +53,8 @@ import {
     faTimes,
     faUserTie,
     faWindowRestore,
-    faFingerprint
+    faFingerprint,
```

Comment:
We've been attempting to keep these in alphabetical order to make them easier to find

Reply:
reordered!